app/pages/lab/export-workflows-dialog.jsx

```diff
@@ -36,17 +42,25 @@ class ExportWorkflowsDialog extends React.Component {
     getWorkflowsInOrder(project, { fields: 'display_name' })
       .then((workflows) => {
         this.setState({ workflows, loading: false });
-      });
-  }
 
-  toggleExport() {
-    if (this.workflowList.selectedIndex >= 0) {
-      this.setState({ workflowSelected: true });
-    }
+        // TODO: load the exports for each workflow
+        workflows.forEach((wf) => {
+          wf.get('classifications_export')
+            .then((media) => {
+              const mediaState = this.state.media;
+              mediaState[wf.id.toString()] = media[0];
+              this.setState({ media: mediaState });
+            })
+            .catch(() => {
+              // this is gonna happen, oh well
```

Comment:
That's a good idea--even though that's what would happen by default anyways, it's a bit more self-documenting.

And you're right, it might also be nice to take a little load off the API to get one request (for the project with its associated export media) instead of five.